src/labeler.cpp

```diff
@@ -4,13 +4,36 @@
 
 #include <glow/GlCapabilities.h>
 
+void handleArgs(int argc, char** argv, Mainframe* frame) {
```

Comment:
Would be better to return the args; maybe as as `std::map<std::string, std::string>`` and then using ``contains("<argument>")`` to check if an option is there outside. (there is also something like boost program_options, etc. that might be a more general version.

```suggestion
std::map<std::string, std::string> parseArgs(int argc, char** argv) {
   std::map<std::string, std::string> parsedArgs;
```

Reply:
Ok. I agree. It's a tad coupled at the moment. Thanks for the feedback. Will take a look.

Reply:
I've decoupled. I used a map as suggested. I did not look at boost program_options. If that's important to you now. I sort of get it. Since it might make extensions by others more intuitive. What do you think?